fdbserver/DataDistribution.actor.cpp

```diff
@@ -2824,6 +2823,21 @@ ACTOR Future<Void> scheduleAuditOnRange(Reference<DataDistributor> self,
 								req.targetServers.push_back(it->second[i].id());
 								storageServersToCheck.push_back(it->second[idx]);
 							}
+							++it;
+							for (; it != rangeLocations[rangeLocationIndex].servers.end(); ++it) {
```

Comment:
why adding this loop, since there is already a loop above?

Reply:
The first loop is going through all servers of the primary DC and selecting a server to do the audit task and adding other servers to compare with the selected server. The second loop goes through all remote DCs and add all servers to compare with the server selected in the first loop.